frontend/lib/src/components/widgets/DataFrame/columns/utils.test.ts

```diff
@@ -533,14 +533,15 @@ describe("truncateDecimals", () => {
 
 describe("formatMoment", () => {
   beforeAll(() => {
-    jest.useFakeTimers("modern")
-    jest.setSystemTime(new Date("2022-04-28T00:00:00Z"))
+    const d = new Date("2022-04-28T00:00:00Z")
+    jest.useFakeTimers()
+    jest.setSystemTime(d)
     timezoneMock.register("UTC")
   })
 
   afterAll(() => {
-    jest.useRealTimers()
     timezoneMock.unregister()
+    jest.useRealTimers()
```

Comment:
Yes. jest is vi under the hood. I'll update these since the lines are already changed. Just don't want to change them all in this PR. I want this PR to focus on the transition where possible.

Reply:
ok, makes sense to do that in a follow up 👍